src/Softeq.NetKit.Chat.Web/TransportModels/Request/Channel/CreateDirectChannelRequest.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace FQ.DEEMA.Chat.Web.TransportModels.Request.Channel
```

Comment:
Why are we using this namespace part? - "FQ.DEEMA."
Let's find all such namespaces and correct it.

Reply:
done